sphinx-doc/features.rst

```diff
@@ -0,0 +1,139 @@
+sampling).
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/04-Custom-Actions-In-Python/00-index`
+
+CPU and GPU devices
+-------------------
+
+HOOMD-blue can execute simulations on CPUs or GPUs. Typical simulations run more efficiently on
+GPUs for system sizes larger than a few thousand particles, although this strongly depends on the
+details of the simulation. The provided binaries support NVIDIA GPUs. Build from source to enable
+preliminary support for AMD GPUs. CPU support is always enabled. GPU support must be enabled at
+compile time with the ``ENABLE_GPU`` CMake option (see :doc:`building`). Select the device to use at
+run time with the `device <hoomd.device>` module. Unless otherwise stated in the documentation,
+**all** operations and methods support GPU execution. At runtime, `hoomd.version.gpu_enabled` indicates
+whether the build supports run time compilation.
+
+MPI
+---
+
+HOOMD-blue can use the message passing interface (MPI) to execute simulations in less time using
+more than one CPU core or GPU. Unless otherwise stated in the documentation, **all** operations and
+methods support MPI parallel execution. MPI support is optional, requires a compatible MPI library,
+and must be enabled at compile time with the ``ENABLE_MPI`` CMake option (see :doc:`building`).
+At runtime, `hoomd.version.mpi_enabled` indicates whether the build supports MPI.
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/03-Parallel-Simulations-With-MPI/00-index`
+
+Threading
+---------
+
+Some operations in HOOMD-blue can use multiple CPU threads in a single process. Control this with
+the `device.Device.num_cpu_threads` property. In this release, threading support in HOOMD-blue is
+very limited and only applies to implicit depletants in `hpmc.integrate.HPMCIntegrator`, and
+`hpmc.pair.user.CPPPotentialUnion`. Threading must must be enabled at compile time with the
+``ENABLE_TBB`` CMake option (see :doc:`building`). At runtime, `hoomd.version.tbb_enabled` indicates
+whether the build supports MPI.
+
+Run time compilation
+--------------------
+
+Some operations allow the user to provide arbitrary C++ code which HOOMD-blue compiles at run time
+and executes during the simulation. `hpmc.pair.user` and `hpmc.external.user` enable users to apply
+arbiratray pair and external potentials to particles in HPMC simulations. `hpmc.pair.user` is
+supports both CPUs and NVIDIA GPUs while `hpmc.external.user` is only supports CPUs. Run time
+compilation must be enabled at compile time with the ``ENABLE_LLVM`` CMake option (see
+:doc:`building`). At runtime, `hoomd.version.llvm_enabled` indicates whether the build supports run
+time compilation.
+
+Mixed precision
+---------------
+
+HOOMD-blue performs computations with mixed floating point precision. There is a **high precision**
+type and a **reduced precision** type. All particle properties are stored in the **high precision**
```

Comment:
I would not bold the high/reduced precision terms after they are first mentioned. It provides too much emphasis. Perhaps use a code syntax style?

Reply:
I just removed the bold.